miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditBoxController.java

```diff
@@ -171,9 +192,43 @@ public String processSubmit(@ModelAttribute("box") Box box, ModelMap model, Sess
       Box original = boxService.get(box.getId());
       box.setBoxables(original.getBoxables());
     }
+    log.info("" + box.getStorageLocation().getId());
     boxService.save(box);
     session.setComplete();
     model.clear();
     return "redirect:/miso/box/" + box.getId();
   }
+
+  private final class BulkCreateBoxBackend extends BulkCreateTableBackend<BoxDto> {
+    public BulkCreateBoxBackend(Class<? extends BoxDto> dtoClass, BoxDto dto, Integer quantity) {
+      super("box", dtoClass, "Boxes", dto, quantity);
+    }
+
+    @Override
+    protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) throws IOException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)